lib/src/main/kotlin/org/promethist/core/model/Turn.kt

```diff
@@ -31,6 +31,8 @@ data class Turn(
 ) : TimeEntity<Turn> {
     data class Request(var attributes: PropertyMap? = null)
 
+    val sessionId get() = session_id // Compatibility with hash key in DynamoDB named sessionId
```

Comment:
I wanted to avoid it. The problem is that sessionId is the key in the turn table and it cannot be renamed. Therefore it requires creating a new table with the updated key name. However the old implementation will stop working with the new key name so it needs to be done simultaneously.

Reply:
Let's use Jackson mixin for this.